src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String supplyName = "supply";
+    private final String buyName = "buy";
+    private int supplySum = 0;
+    private int baySum = 0;
+    private int result = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFile(fromFileName);
+        writeToFile(toFileName);
+    }
+
+    private String readFile(String fromFileName) {
```

Comment:
It is better to divide this method into two separate ones. Reading from a file and creating a report.